pkg/config/lvmd/lvmd.go

```diff
@@ -133,8 +133,20 @@ func NewLvmdConfigFromFile(p string) (*Lvmd, error) {
 	return l, nil
 }
 
-func LvmSupported() error {
-	if _, err := exec.LookPath("vgs"); err != nil {
+func SaveLvmdConfigToFile(l *Lvmd, p string) error {
```

Comment:
Just to persist what we discussed.
I think this file gets `p = /etc/microshift/lvmd.yaml` and that can be a problem in scenario such as:
1. User creates ostree commit without specifying `/etc/microshift/lvmd.yaml`
2. MicroShift starts, generates default file in `/etc/microshift/lvmd.yaml`
3. System works, everything is fine
4. User creates NEW ostree commit and specifies `/etc/microshift/lvmd.yaml`
5. User deploys the commit
6. OSTree performs its 3-way merge on /etc ([ref](https://containers.github.io/bootc/filesystem.html#etc)) and the result is that `the default file that microshift created` is not overwritten by `user supplied lvmd.yaml` thus breaking the possibility to update lvmd configuration using ostree.

In such case we need to avoid writing in /etc to files that user might edit.
What we came up with is:
1. setup an inotify on `/etc/microshift/lvmd.yaml`
2. if `/etc/microshift/lvmd.yaml` changes, copy it to `/var/lib/microshift/*/lvmd.yaml`
3. mount `/var/lib/microshift/*` for lvms

That way we still get hot reload of the file, but we don't mess up the /etc updates.